iosApp/Survey/Sources/Presentation/Modules/SurveyDetail/SurveyQuestion/QuestionPickerView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  QuestionPickerView.swift
+//  Survey
+//
+//  Created by Bliss on 11/1/23.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct QuestionPickerView: View {
+
+    @State private var selectedInput = ""
+
+    let ids: [String]
+
+    var body: some View {
+        Picker(String(describing: Self.self), selection: $selectedInput) {
+            ForEach(ids, id: \.self) { id in
+                let font = selectedInput == id ? Font.boldLarge : Font.regularLarge
+                Text(id)
+                    .font(font)
+                    .foregroundColor(Color.white)
+                if id != ids.last {
+                    Rectangle()
```

Comment:
[Optional] I think we can use [Divider](https://developer.apple.com/documentation/swiftui/divider) here